src/components/preset-control/preset-delete-dialog/preset-delete-dialog.tsx

```diff
@@ -0,0 +1,43 @@
+import { ReactElement } from 'react'
+
+import { t } from '@i18n'
+import { Button } from '@ui/button'
+import { Dialog } from '@ui/dialog'
+
+interface IPresetDeleteDialog {
+  isOpen?: boolean
+  onClose: () => void
+  onDelete: () => void
+  presetName: string
+}
+
+export const PresetDeleteDialog = ({
+  isOpen,
+  onClose,
+  onDelete,
+  presetName,
+}: IPresetDeleteDialog): ReactElement => {
+  return (
+    <Dialog
+      isOpen={isOpen}
+      onClose={onClose}
+      title={t('presetControl.deleteDialog.title')}
+      actions={
```

Comment:
Let's try to adjust PopupCard component to work with Modal, it will include the footer with buttons and the header from your Dialog component (starts on 42nd line).

Reply:
Created `DialogCard` and `DialogStandardActions`